src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -731,26 +731,32 @@ namespace AppInstaller::CLI::Workflow
                 if (updateAvailable || !m_onlyShowUpgrades)
                 {
                     Utility::LocIndString availableVersion, sourceName;
+                    Utility::LocIndString packageId = match.Package->GetProperty(PackageProperty::Id);

```

Comment:
Getting value into `packageId` but not using it.

Reply:
Resolved in latest commit. Oops!